src/EPR.Accreditation.API/Controllers/AccreditationController.cs

```diff
@@ -98,6 +98,18 @@ public async Task<IActionResult> CreateAccreditation([FromBody] DTO.Accreditatio
 
             return Ok();
         }
+
+        [HttpPut("{id}/HasOverseasAgent")]
+        public async Task<IActionResult> HasOverseasAgent(
```

Comment:
We shouldn't have this end point - it's too specific. To identify if we have an overseas agent we should use the get accreditation end point which has that property on.